WikiWatershed/model-my-watershed
# Integrate sidebar (updated with style fixes)

Supersedes #1105 

Testing instructions:
* Select an area, run models, and report any styling or functionality issues.

Known issues: 
* Some model runs appear to be canceling each other out, but all the results appear to come back. It's not clear to me yet what's going on here. It happened more frequently when I switched models to run only  when that scenario's model tab was activated. To avoid these problems, I'm running all of the models at once when the page is loaded (which is what we are doing now).

Connects to #1093 
Connects to #1117
